test/cqlpy/conftest.py

```diff
@@ -272,4 +277,4 @@ def compact_storage(cql):
         # enable_create_table_with_compact_storage is a scylla only feature
         # so the above may fail on cassandra.
         # This is fine since compact storage is enabled there by default.
-        pass
+        yield None
```

Comment:
The comment is outdated, no? compact storage is gone in Cassandra.

Reply:
And what does "yield None" do?

Reply:
Are you sure it's "gone in Cassadra"? In which version? I just ran this test against Cassandra 5.0.0 and it passes. It isn't seem to be gone, the test is passing. Making it gone was **our** decision. In fact, I'm not even sure it was a decision - see https://github.com/scylladb/scylladb/issues/12263#issuecomment-1853793493 and that fact that that issue isn't even closed :-( Looking at my earlier comments in that issue, I was reminded that Cassandra indeed re-implemented COMPACT STORAGE at some point, but I believe it was never removed.

About the yield - a fixture is supposed to yield something which is saved as the value of the fixture. Here I have no value. I could have just written "yield" without this None, it would have done the same.

Reply:
You're right, I misremembered.

Reply:
Since I need to send a new version anyway (I botched out something that CI discovered), I'll remove the unnessary "None" from the yield command.